examples/initExternal/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
I'll skip the examples for this first review round and focus on the SDK code since they will need adjusting.